tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/DynamoDb/AwsDynamoDbCommon.cs

```diff
@@ -87,5 +88,25 @@ public static void TagBatchRequest<TBatchRequest>(TBatchRequest request, AwsDyna
                 TagTableNameAndResourceName(tableName, tags, scope);
             }
         }
+
+        public static string GetValueFromDynamoDbAttribute(IDynamoDbAttributeValue value)
+        {
+            if (value.S != null)
```

Comment:
There's a compiler warning about `IDynamoDbAttributeValue.S` not being nullable, so it thinks this condition is always true. You probably want to declare `S` and `N` as `string?` in the dock type interface.